.github/workflows/testing.yml

```diff
@@ -478,6 +478,15 @@ jobs:
   build_and_test_windows:
     name: Build and Test Windows
     needs: check_files
+    strategy:
+      matrix:
+        include:
+          - os: windows-2019
```

Comment:
There's already a CPU task labelled `build_and_test_cpu_windows` which tests LLVM15, what's the difference with this one?

Reply:
changed to cuda.